server/upstream/upstream-tarballs.txt

```diff
@@ -646,6 +646,7 @@ stk:httpls:https://ccrma.stanford.edu/software/stk/download.html
 sunpinyin:google:sunpinyin
 swfdec-mozilla:subdirhttpls:http://swfdec.freedesktop.org/download/swfdec-mozilla/
 swfdec:subdirhttpls:http://swfdec.freedesktop.org/download/swfdec/
+swig:sf:swig
```

Comment:
On Mon, 2015-02-16 at 06:11 -0800, Vincent Untz wrote:
> In server/upstream/upstream-tarballs.txt:
> 
> > @@ -646,6 +646,7 @@ stk:httpls:https://ccrma.stanford.edu/software/stk/download.html
> >  sunpinyin:google:sunpinyin
> >  swfdec-mozilla:subdirhttpls:http://swfdec.freedesktop.org/download/swfdec-mozilla/
> >  swfdec:subdirhttpls:http://swfdec.freedesktop.org/download/swfdec/
> > +swig:sf:swig
> 
> Unfortunately, that doesn't work. You can test this
> with: ./download-upstream-versions --debug "swig:sf:swig"
> 
> It might be that the sf method is broken nowadays, and
> download-upstream-versions would need fixing...
> 
> —
Yes, sourceforge changed everything again, and I'm not entirely certain
how to fix that.   Let me have a look.